### PR TITLE
refactor(core): Remove linting exceptions in nodes-base

### PR DESCRIPTION
**Pull request by michael-radency**

### CODE REVIEW

File: packages/nodes-base/.eslintrc.js [hunk 1]
```diff
@@ -55,7 +55,8 @@ module.exports = {
 		'@typescript-eslint/restrict-template-expressions': 'off',
 		'@typescript-eslint/return-await': 'off',
 		'@typescript-eslint/unbound-method': 'off',
-		'@typescript-eslint/ban-ts-comment': ['warn', { 'ts-ignore': true }],
+		// '@typescript-eslint/ban-ts-comment': ['warn', { 'ts-ignore': true }],
+		'@typescript-eslint/ban-ts-comment': 'off',
```
Comment (netroy) [inline packages/nodes-base/.eslintrc.js:59]: we shouldn't be turning this rule off, as that will only encourage the use of `@ts-ignore`, which defeats the purpose of using a typed language.

Comment (michael-radency) [reply packages/nodes-base/.eslintrc.js:59]: it is temporary, I will re enable it after completion

Comment (netroy) [reply packages/nodes-base/.eslintrc.js:59]: ah okay. in that case please mark the PR as Draft. 👍🏽

File: packages/nodes-base/nodes/Box/Box.node.ts [hunk 2]
```diff
@@ -187,9 +187,9 @@ export class Box implements INodeType {
 							const createdRangeValues = (additionalFields.createdRangeUi as IDataObject)
 								.createdRangeValuesUi as IDataObject;
 							if (createdRangeValues) {
-								qs.created_at_range = `${moment
-									.tz(createdRangeValues.from, timezone)
-									.format()},${moment.tz(createdRangeValues.to, timezone).format()}`;
+								qs.created_at_range = `${moment.tz(createdRangeValues.from, tz).format()},${moment
+									.tz(createdRangeValues.to, tz)
+									.format()}`;
```
Comment (netroy) [inline packages/nodes-base/nodes/Box/Box.node.ts:192]: this formatting still looks not very readable. maybe we should extract these into variables like this:
```suggestion
								const start = moment.tz(createdRangeValues.from, tz).format();
								const ended = moment.tz(createdRangeValues.to, tz).format()
								qs.created_at_range = `${start},${ended}`;
```

**Review by netroy — APPROVED** [review]
Thanks for doing this. This will definitely make a lot of us happier. 🙏🏽 
PS: I also added some commits made via `eslint --fix`

**Comment by janober** [comment]
Got released with `n8n@0.206.1`

**Pull request merged**